ChangeLog

```diff
@@ -23,6 +23,9 @@ What's New in astroid 2.14.2?
 =============================
 Release date: TBA
 
+* '_infer_str_format_call' won't crash anymore when the string it analyses are uninferable.
```

Comment:
Let's just release `2.15` if we want to. There are fixes in there that are worth bringing out sooner than later imo.

Reply:
There's the group exception for 3.11 that need to be done before releasing. I'm planning to try to do that next.